tests/unit/resource/shutdown_suspended_pus.cpp

```diff
@@ -75,7 +75,7 @@ int main(int argc, char* argv[])
                 hpx::resource::scheduling_policy::local_priority_fifo,
                 hpx::resource::scheduling_policy::local_priority_lifo,
```

Comment:
I think `local_priority_lifo` needs to be `#ifdef`ed away like the ABP scheduler, no? Same in the other tests.

Reply:
Uhh, I missed those. Thanks!